.circleci/config.yml

```diff
@@ -12,10 +12,15 @@ jobs:
     steps:
       - checkout
       - node/install:
-          install-yarn: true
           node-version: "20.10"
+      - run:
+          name: Use yarn berry
+          command: |
+            corepack enable && yarn set version berry
```

Comment:
using similar "recommended" strategy of specific run block for circleci's orb w/yarn berry as discussed here: https://discuss.circleci.com/t/how-do-i-use-yarn-3-4-1-with-the-node-orb/47187/2